src/resolvers.js

```diff
@@ -4,42 +4,92 @@
 import apiFetch from '@wordpress/api-fetch';
 
 export async function fetchThemeJson() {
-	const fetchOptions = {
+	return apiFetch( {
 		path: '/create-block-theme/v1/get-theme-data',
 		method: 'GET',
 		headers: {
 			'Content-Type': 'application/json',
 		},
-	};
-
-	try {
-		const response = await apiFetch( fetchOptions );
-
+	} ).then( ( response ) => {
 		if ( ! response?.data || 'SUCCESS' !== response?.status ) {
 			throw new Error(
 				`Failed to fetch theme data: ${
 					response?.message || response?.status
 				}`
 			);
 		}
-
 		return JSON.stringify( response?.data, null, 2 );
-	} catch ( e ) {
-		// @todo: handle error
-	}
```

Comment:
Nitpick I guess, but prefer async/await instead of `.then`.

Reply:
That's the PHP programmer in you.  .then .catch can be a lovely thing. :)

Reply:
> That's the PHP programmer in you. .then .catch can be a lovely thing. :)

It's mostly about keeping a consistent codebase, since we use async/await on most other places.

It's also a bit pointless to have an `async function` that does not use `await` within; in this case, `apiFetch` is already returning a Promise.